docs/designs/PowerShell-AzF-Overall-Design.md

```diff
@@ -67,6 +67,19 @@ The host process sends an _InvocationRequest_ every time a function is triggered
 
 The real invocation happens mainly in the PowerShell Manager. It transforms the input binding data to the appropriate types that are easy to work with in PowerShell, and then invokes the function script with the transformed value as arguments. Once the invocation finishes, it collects the results for the output bindings. The output results will then be converted to the pre-defined gRPC data type, and sent to the Host within an _InvocationResponse_.
 
+#### TraceContext
+
+As part of the _InvocationRequest_, the host also sends TraceContext which contains the Traceparent (the current Activity id), Tracestate and Attributes (The current map of tags from Activity). Users can access this by specifying it as parameter as shown in the example below.
+
+```powershell
+
+param($req, $TraceContext)
+
+$activityId = $TraceContext.Traceparent
```

Comment:
Is there any specific tracing facility that the customer is supposed to be using with this data, or this is entirely up to the customer? I'm thinking about an code sample that would demonstrate the actual usage, not just retrieving properties into variables, e.g. `Write-Verbose "[$($TraceContext.Traceparent)] My message"`.

Reply:
Update text and pointed to https://www.w3.org/TR/trace-context/ as discussed